app/lib/dns.server.ts

```diff
@@ -41,33 +56,21 @@ export const createHostedZone = async (domain: string) => {
     }
     return response.HostedZone.Id.replace('/hostedzone/', '');
   } catch (error) {
-    // TODO: fix with better logging...
-    console.warn(error);
+    logger.error(error);
```

Comment:
Reserve `error` for fatal errors, and use `warn` for situations that we can recover from (like these).

Reply:
Will be updated